From f9fddf1446c8d89cac8222d9d444204bc8bee8d1 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Sun, 1 Sep 2013 14:45:44 +0200
Subject: [PATCH] remove cmupdater (not needed for unofficial builds)

---
 config/common.mk |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index a204f9b..fe4099f 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -164,7 +164,6 @@ PRODUCT_PACKAGES += \
     audio_effects.conf \
     CMWallpapers \
     Apollo \
-    CMUpdater \
     CMFileManager \
     LockClock \
     CMAccount
-- 
1.7.9

