From 187e9a3be7de3ab6cdea235ddce163929db3909d Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Tue, 3 Sep 2013 00:51:09 +0200
Subject: [PATCH] mako hdpi build

---
 device.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/device.mk b/device.mk
index 6b33e34..b1d6843 100644
--- a/device.mk
+++ b/device.mk
@@ -25,8 +25,8 @@ DEVICE_PACKAGE_OVERLAYS := device/lge/mako/overlay
 # currently contain all of the bitmaps at xhdpi density so
 # we do this little trick to fall back to the hdpi version
 # if the xhdpi doesn't exist.
-PRODUCT_AAPT_CONFIG := normal hdpi xhdpi
-PRODUCT_AAPT_PREF_CONFIG := xhdpi
+PRODUCT_AAPT_CONFIG := normal hdpi
+PRODUCT_AAPT_PREF_CONFIG := hdpi
 
 PRODUCT_PACKAGES := \
 	lights.mako
@@ -128,7 +128,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 	ro.opengles.version=196608
 
 PRODUCT_PROPERTY_OVERRIDES += \
-	ro.sf.lcd_density=320
+	ro.sf.lcd_density=240
 
 # Audio Configuration
 # FIXME: Remove persist.audio.handset.mic and persist.audio.fluence.mode
-- 
1.8.1.2

