From 7898d8e7a5ed211bc1d6b86bc4fb8e02f2e2e3b6 Mon Sep 17 00:00:00 2001
From: Micha LaQua <micha.laqua@gmail.com>
Date: Tue, 12 Nov 2013 23:28:53 +0100
Subject: [PATCH] mako hdpi build

---
 device.mk |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/device.mk b/device.mk
index 28ea228..9a6f019 100644
--- a/device.mk
+++ b/device.mk
@@ -25,8 +25,8 @@ DEVICE_PACKAGE_OVERLAYS := device/lge/mako/overlay
 # currently contain all of the bitmaps at xhdpi density so
 # we do this little trick to fall back to the hdpi version
 # if the xhdpi doesn't exist.
-PRODUCT_AAPT_CONFIG := normal hdpi xhdpi
-PRODUCT_AAPT_PREF_CONFIG := xhdpi
+PRODUCT_AAPT_CONFIG := normal hdpi
+PRODUCT_AAPT_PREF_CONFIG := hdpi
 
 PRODUCT_PACKAGES := \
 	lights.mako
@@ -125,7 +125,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 	ro.opengles.version=196608
 
 PRODUCT_PROPERTY_OVERRIDES += \
-	ro.sf.lcd_density=320
+	ro.sf.lcd_density=240
 
 # Audio Configuration
 # FIXME: Remove persist.audio.handset.mic and persist.audio.fluence.mode
-- 
1.7.9

