From 47575fa22c1ad571df91ea0bd9af8bde5da06d77 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Mon, 28 Oct 2013 11:14:14 +0100
Subject: [PATCH] remove cmupdater (not needed for unofficial builds)

---
 config/common.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 1b8a5e2..8e69064 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -207,7 +207,6 @@ PRODUCT_PACKAGES += \
 ifneq ($(TARGET_BUILD_VARIANT),user)
 
 PRODUCT_PACKAGES += \
-    CMUpdater \
     Superuser \
     su
 
-- 
1.8.1.2

