From c72b4c0e9a4fce6fede744035de596ac70004808 Mon Sep 17 00:00:00 2001
From: milaq <micha.laqua@gmail.com>
Date: Sat, 7 Dec 2013 12:40:05 +0100
Subject: [PATCH] hammerhead hdpi build

---
 device.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/device.mk b/device.mk
index 9262a16..ba385c5 100644
--- a/device.mk
+++ b/device.mk
@@ -94,8 +94,8 @@ PRODUCT_TAGS += dalvik.gc.type-precise
 # currently contain all of the bitmaps at xhdpi density so
 # we do this little trick to fall back to the hdpi version
 # if the xhdpi doesn't exist.
-PRODUCT_AAPT_CONFIG := normal hdpi xhdpi xxhdpi
-PRODUCT_AAPT_PREF_CONFIG := xxhdpi
+PRODUCT_AAPT_CONFIG := normal hdpi xhdpi
+PRODUCT_AAPT_PREF_CONFIG := xhdpi
 
 PRODUCT_CHARACTERISTICS := nosdcard
 
@@ -202,7 +202,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
     ro.opengles.version=196608
 
 PRODUCT_PROPERTY_OVERRIDES += \
-    ro.sf.lcd_density=480
+    ro.sf.lcd_density=320
 
 PRODUCT_PROPERTY_OVERRIDES += \
     persist.hwc.mdpcomp.enable=true
-- 
1.8.3.2

